modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -31,133 +31,7 @@ See "Configuring the PTP fast event notifications publisher" for more informatio
 .Recommended PTP boundary clock configuration
```

Comment:
This is a legacy title, but consider changing the title to move away from a recommendation to more of an example:

See https://www.ibm.com/docs/en/ibm-style?topic=information-claims-recommendations

![Screenshot from 2023-09-27 12-38-43](https://github.com/openshift/openshift-docs/assets/57954076/18bd37b2-7c64-4cfa-937d-3327cce9cf29)

Reply:
We'll consider this for future updates thanks.

Reply:
Actually, this change makes sense. I'll update as you suggest :)